docs/asset-store/docs/03-03-minio-gateway.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Minio and Minio Gateway
+type: Details
+---
+
+The whole concept of the Asset Store relies on Minio as a storage solution. It supports the Kyma's manifesto and the "batteries included" rule as Kyma provides you with this on-premise solution by default.
+
+Depending on the usage scenario, you can:
+- Use Minio for local development.
+- Store your assets on a production scale using Minio in a [Gateway mode](https://github.com/minio/minio/tree/master/docs/gateway).
+
+The Asset Store ensures that both usage scenarios work for Kyma, without additional configuration of the in-build controllers.
+
+## Local storage
```

Comment:
maybe naming it `development mode storage`

Reply:
Sure, what not ;)